src/backend/access/aocs/aocsam.c

```diff
@@ -910,6 +911,12 @@ aocs_insert_values(AOCSInsertDesc idesc, Datum *d, bool *null, AOTupleId *aoTupl
 								   RelationGetRelationName(idesc->aoi_rel));	/* tableName */
 #endif
 
+	/*
+	 * generate new transaction id if necessary, so dependent entities, such as
+	 * spgist indexes, can use it outside aocs_insert_values
+	 */
+	GetCurrentTransactionId();
```

Comment:
Sometimes` (void) GetCurrentTransactionId();` is used to show that return value is explicitly ignored.
For example, [here](https://github.com/arenadata/gpdb/blob/adb-6.x/src/backend/utils/cache/relcache.c#L3267) or [here](https://github.com/arenadata/gpdb/blob/adb-6.x/src/backend/commands/async.c#L816).
What do you think?

Reply:
Will be fixed once we'll make a decision about fix place. Thanks.